core-bootstrap/src/components/collapse/common.ts

```diff
@@ -0,0 +1,178 @@
+import {createTransition} from '@agnos-ui/core/services/transitions/baseTransitions';
+import type {ConfigValidator, Directive, PropsConfig, Widget} from '@agnos-ui/core/types';
+import {stateStores, writablesForProps} from '@agnos-ui/core/utils/stores';
+import {bindDirectiveNoArg} from '@agnos-ui/core/utils/directive';
+import {typeBoolean} from '@agnos-ui/core/utils/writables';
+import {collapseHorizontalTransition, collapseVerticalTransition} from '../../services/transitions/collapse';
+import {asWritable, computed} from '@amadeus-it-group/tansu';
+
+export interface CommonCollapseCommonPropsAndState {
```

Comment:
Why did you create this `common.ts` file with the `Common` prefix instead of having all this content directly in collapse.ts without the `Common` prefix?